ci/jenkins/test.sh

```diff
@@ -461,7 +462,23 @@ function deliver_antrea_linux_containerd {
     echo "==== Finish building and delivering Linux containerd images ===="
 }
 
+# Function to check file changes using git
+check_file_change() {
+    local file=$1
+    local bit=$2
+    # Check if the file has been modified
+    if git diff --name-only HEAD | grep -q "$file"; then
+        # Set the corresponding bit in PULL_MASK if the file is modified
+        PULL_MASK=$((PULL_MASK | $bit))
+    fi
+}
+
 function deliver_antrea_windows_containerd {
+    echo "=====Check each base image file and update PULL_MASK accordingly====="
+    check_file_change "build/images/base-windows/Dockerfile" 1  # Corresponds to 0x1
+    check_file_change "build/images/base-windows/Dockerfile" 2  # Corresponds to 0x2
```

Comment:
Why do you use `build/images/base-windows/Dockerfile` for all of 1, 2, 4? Do you forget to update the docker file name?

Reply:
The updates of these three images depend on whether the same file has been modified.